Chaincase.UI/Components/Status.razor

```diff
@@ -1,5 +1,7 @@
 ﻿@inject StatusViewModel StatusViewModel
+@inject BitcoinStore BitcoinStore
```

Comment:
Nyet. Views should not depend on details of the implementation. Move to e.g. `StatusViewModel.SmartHeaderChain`